docs/doc/12-load-data/00-transform/07-data-load-transform.md

```diff
@@ -2,7 +2,23 @@
 title: Transforming Data on Load
 ---
 
-Databend offers a powerful feature that enables data transformation during the loading process using the [COPY INTO](/14-sql-commands/10-dml/dml-copy-into-table.md) command. This functionality simplifies your ETL pipeline by incorporating basic transformations, eliminating the need for temporary tables. By transforming data during loading, you can streamline your ETL process effectively. Here are practical ways to enhance data loading with this feature:
+Databend offers a powerful feature that enables data transformation during the loading process using the [COPY INTO](/14-sql-commands/10-dml/dml-copy-into-table.md) command, with this syntax:
+
+```sql
+COPY INTO [<database>.]<table_name> [ ( <col_name> [ , <col_name> ... ] ) ]
+     FROM ( SELECT [<file_col> ... ]
+            FROM { userStage | internalStage | externalStage } )
+[ FILES = ( '<file_name>' [ , '<file_name>' ] [ , ... ] ) ]
+[ PATTERN = '<regex_pattern>' ]
+[ FILE_FORMAT = ( TYPE = {PARQUET} [ formatTypeOptions ] ) ]
+[ copyOptions ]
+```
+
+:::note
+This feature is currently only available for the Parquet file format.
```

Comment:
we support CSV/Parquet both with: `select $1, $2 ...`

Reply:
https://databend.rs/doc/load-data/transform/querying-stage

Reply:
yes all format support thansform, the select part is the same as  https://databend.rs/doc/load-data/transform/querying-stage